engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -40,17 +40,20 @@
     private RenderTaskListGenerator taskListGenerator; // TODO: investigate ways to remove nodes influence on taskList
     private boolean enabled = true;
 
-    protected FBO requiresFBO(FBOConfig fboConfig, BaseFBOsManager frameBuffersManager) {
+    public void initialise(Object initialData) { } // added here to avoid adding it to all other nodes.
```

Comment:
I am alsi not familiar with that code but introducing a class type parameter T taht initualize takes could potentially allow you to keep typesavety. Another idea that might be applyable or not would be to do the initialization in a node specific factory class that implements a node factory interface.